modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -3310,6 +3310,11 @@ export const console: ConsoleNS = {
                                     required: "Client secret is a required field."
                                 }
                             },
+                            enableGoogleOneTap: {
```

Comment:
We need to add translations for other languages as well.

Reply:
Yes. Will be added in another iteration